rllib/agents/sac/sac_model.py

```diff
@@ -111,17 +110,68 @@ def __init__(self,
         shift_and_log_scale_diag = tf.keras.layers.Concatenate(axis=-1)(
             [shift, log_scale_diag])
 
-        raw_action_distribution = tfp.layers.MultivariateNormalTriL(
-            self.action_dim)(shift_and_log_scale_diag)
+        batch_size = tf.keras.layers.Lambda(lambda x: tf.shape(input=x)[0])(
```

Comment:
Hm what are the changes here for?

Reply:
Replacing distributional computation with more explicit code from softlearning codebase.

Reply:
My policy implementation in softlearning is not ideal and could be refactored to be a lot cleaner, which is why I think we shouldn't bring those changes over here. Instead, it would be better to refactor both of them, but that can be left for future PR.

Reply:
^^ The distributional code was a lot more concise. I switched back to softlearning database to debug via making everything deterministic.